docs/admin-guide/install-buildout.md

```diff
@@ -75,21 +77,28 @@ eggs =
     Plone
 ```
 
+Install Buildout script in bin directory.
+
+```shell
+./venv/bin/buildout bootstrap
+```
+
 Run Buildout.
 
 ```shell
-bin/buildout
+./bin/buildout
```

Comment:
Why? It's more typing and unnecessary.

Reply:
in my experience it works better than without. Sometimes on Linux calling a script without `./` will not find it.
Thats why i always use it, but I'm ok if we leave this out.

Reply:
I don't feel strongly about it, but I don't think I ever had problems with this.

Reply:
I think it is only required when trying to run a script in the current working directory, but not a binary. In other words, the following example fails:

```
> script
```

Whereas the following will succeed.

```
> ./script
```

However, as long as a path to the script is specified—whether that is `./script`, `dir1/script`, or `dir1/dir2/script`—then it will be invoked.

More [discussion from SO](https://stackoverflow.com/a/6331085/2214933) explains how `/` and `$PATH` work together.

As far as in this example, I would remove `./` because there is a `/` in `bin/buildout`, making `./` optional.